app/controllers/api/v1/massages_controller.rb

```diff
@@ -0,0 +1,9 @@
+# app/controllers/api/massages_controller.rb
+module Api
+  class Api::V1::MassagesController < ApplicationController
+    def random_massage
+      @greeting = Message.order('RANDOM()').first
+      render json: { message: @greeting.message }
+    end
```

Comment:
@- [ ] Kindly make sure you change the `random_massage` method name to `index` as it shows the below error when trying to test your endpoint to display random messages(.i.e. `http://localhost:3000/api/v1/massages`).
![Screenshot from 2023-04-05 21-34-12](https://user-images.githubusercontent.com/83042742/230250435-9b0ce666-65e4-4b88-b548-63a7abd928d8.png)